### PR TITLE
Upgrade from haskell-lsp(-types) to lsp(-types)

### PR DESCRIPTION
**Pull request by cocreature**

### CODE REVIEW

File: src/Development/IDE/Plugin/Completions/Logic.hs [hunk 1]
```diff
@@ -189,27 +204,27 @@ mkModCompl :: T.Text -> CompletionItem
 mkModCompl label =
   CompletionItem label (Just CiModule) Nothing Nothing
     Nothing Nothing Nothing Nothing Nothing Nothing Nothing
-    Nothing Nothing Nothing Nothing Nothing
+    Nothing Nothing Nothing Nothing Nothing Nothing
```
Comment (sofiafaro-da) [inline src/Development/IDE/Plugin/Completions/Logic.hs]: Can we use a pattern like `(defaultCompletionItem label) { _kind = Just CiModule  }` for here and the others?

**Review by sofiafaro-da — APPROVED** [review]
Great work, thank you!

File: src/Development/IDE/Plugin/Completions/Logic.hs (hunk 1 above)
Comment (cocreature) [reply src/Development/IDE/Plugin/Completions/Logic.hs]: good idea a5011720e7f7dfa2d5a41c58446ba3402f4a15a7

Comment (sofiafaro-da) [reply src/Development/IDE/Plugin/Completions/Logic.hs]: Thanks!

**Review by ghost — COMMENTED** [review]
Thanks!

**Pull request merged**